feature-matcher-generator-e2e-test/src/test/java/ru/yandex/qatools/proc/test/MobTest.java

```diff
@@ -0,0 +1,61 @@
+package ru.yandex.qatools.proc.test;
+
+import org.hamcrest.Matcher;
+import org.junit.Test;
+
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+/**
+ * @author Vladislav Bauer
+ */
+
+public class MobTest {
```

Comment:
think it can be converted to parameterized test with list of expected matchers. (the name of matchers itself gets enough info about type)